### PR TITLE
TF2 scripts

### PR DESCRIPTION
**Pull request by c0redumb**
Updated the run scripts to use TF2 containers
Added script to measure performance
Updated README

### CODE REVIEW

**Comment by c0redumb** [comment]
Diego, I think that image is public. I can see it here. https://hub.docker.com/r/rocm/tensorflow/tags. It is not public when you pull it?

File: scripts/calc_performance_metrics.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            val = float(l.split(' ')[-1])
+            sum += val
+            count += 1
+        #print(sum, count)
+        print("Average global steps per second: {:.3f}".format(sum / count))
+        print("Average tokens per second: {:.2f}".format(sum / count * seq_len * batch_size * ngpus))
+        print("    given sequence length {}, batch size {} and {} GPU(s) used".format(seq_len, batch_size, ngpus))
+except:
+    #traceback.print_exc()
+    print("Error opeing run_output file", sys.argv[1])
```
Comment (micmelesse) [inline scripts/calc_performance_metrics.py:37]: Typo

File: scripts/calc_performance_metrics.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ngpus = 1
+if len(sys.argv) > 4:
+    ngpus = int(sys.argv[4])
+
+try:
+    with open(sys.argv[1]) as f:
+        sum = 0
+        count = 0
+        for l in f.readlines():
+            if not "global_step/sec" in l:
```
Comment (micmelesse) [inline scripts/calc_performance_metrics.py:24]: people usually use regex to match lines

File: scripts/calc_performance_metrics.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+                continue
+            if not "tensorflow" in l:
+                continue
+            val = float(l.split(' ')[-1])
+            sum += val
+            count += 1
+        #print(sum, count)
+        print("Average global steps per second: {:.3f}".format(sum / count))
+        print("Average tokens per second: {:.2f}".format(sum / count * seq_len * batch_size * ngpus))
```
Comment (micmelesse) [inline scripts/calc_performance_metrics.py:33]: Should you not be using the examples per second to calculate the tokens per second?

File: scripts/calc_performance_metrics.py (hunk 2 above)
Comment (micmelesse) [reply scripts/calc_performance_metrics.py:24]: But it is fine if it works

Comment (c0redumb) [reply scripts/calc_performance_metrics.py:24]: Never liked Regex. This is more pythonian. :-) I will fix the typo and move it in.

**Pull request merged**